npm/react/src/mount.ts

```diff
@@ -47,6 +62,10 @@ export function mount (jsx: React.ReactNode, options: MountOptions = {}, rerende
   return makeMountFn('mount', jsx, { ReactDom: ReactDOM, ...options }, rerenderKey, internalOptions)
 }
 
+/**
+ * Unmounts the component from the DOM.
+ * @param options - Options for unmounting.
+ */
 export function unmount (options = { log: true }) {
```

Comment:
If this function sticks around we should make a type for the options and use that.

Reply:
I've also seen `@internal` but I cannot see that documented in jsdoc.

If we drop it, we could mark as `@deprecated`. Some IDEs will put a line through it.

![image](https://user-images.githubusercontent.com/19196536/199133515-d45d3ec7-b456-4219-8908-1172e535094c.png)

Reply:
I've left this one alone in hopes we handle it in https://github.com/cypress-io/cypress/pull/24419